squidowl/halloy
# Reload config in an async manner

Fixes #551
Needed for #552, #424

Test plan:
1) open halloy
2) test reloading config
3) apply patches in #552
4) observe that UI operates while long-running command is blocking the
   config from reloading
